.scripts/commit-rule

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
@goonerDroid Can you remove this for now? I will add this rule via GitHub settings

Reply:
@goonerDroid Ack. Will remove `commit-rule`(lock commits on the main branch)